packages/app-store/googlecalendar/lib/CalendarService.ts

```diff
@@ -97,10 +97,9 @@ export default class GoogleCalendarService implements Calendar {
         responseStatus: "accepted",
       })) || [];
     return new Promise(async (resolve, reject) => {
-      const [mainHostDestinationCalendar] =
-        calEventRaw?.destinationCalendar && calEventRaw?.destinationCalendar.length > 0
-          ? calEventRaw.destinationCalendar
-          : [];
+      const selectedHostDestinationCalendar = calEventRaw.destinationCalendar?.find(
+        (cal) => cal.credentialId === credentialId
+      );
```

Comment:
Was creating the event but always sending mainHostAttendee so it will eventually get 3 calendar event entries.